tests/watcher.test.js

```diff
@@ -4,9 +4,23 @@
 
 const { ChainOps } = require('../dist/index')
 
-describe('ChainOps Watcher API', () => {
+describe.only('ChainOps Watcher API', () => {
```

Comment:
remove cheeky `.only`

Reply:
oops. good catch, the linter isn't catching this because i'm only running it on the `*.ts` files. i'll run a linter on the `*.js` tests too